administrator-manual/en/disaster_recovery.rst

```diff
@@ -165,3 +168,102 @@ the configuration at boot-time, to ensure a minimal network
 connectivity and login again on the Server Manager.
 
 
+.. _skip-network-restore-section:
+
+Skip network restore
+--------------------
+
+Network configuration is restored by default, but sometimes it is necessary to restore an 
+installation on a different hardware without migrating the network configuration.
+This is a common scenario when moving a virtual machine from a VPS provider to another.
+
+To disable the network restore, make sure to disable the :guilabel:`Restore network configuration` option from
+the new Server Manager.
+
+Since some application configurations depend on network interface names, not everything can be automatically restored.
+
+DHCP
+^^^^
+
+DHCP servers on non-existing interfaces will be deleted.
+If needed, please reconfigure the DHCP from the Server Manager.
+See also :ref:`dhcp-section` for more general information.
+
+Samba Active Directory
+^^^^^^^^^^^^^^^^^^^^^^
+
+Samba Active Directory requires a network bridge in the green zone for the local running container.
+If the bridge already exists, the container will continue running after the restore.
+
+If the bridge does not exist anymore, Samba Active Directory is stopped.
+To enable it again:
+
```

Comment:
```suggestion
- make sure the old IP address of the container is still inside the new green network. If not, set it from command line, eg: ::
    config setprop nsdc bridge IpAddress 192.168.1.2
```

Reply:
I want to make some tests: I'm not sure the nethserver-dc-save event is good. We may use nethserver-dc-change-ip instead.